flasher/nina.go

```diff
@@ -86,19 +89,22 @@ func (f *NinaFlasher) FlashFirmware(firmwareFile *paths.Path) error {
 	}
 
 	logrus.Debugf("Checking md5")
-	err = f.md5sum(data)
-	if err != nil {
+	if err := f.md5sum(data); err != nil {
 		logrus.Error(err)
 		return err
 	}
-	logrus.Debugf("Flashed all the things")
-	return nil
+	logrus.Infof("Flashed all the things")
+	flasherOut.Write([]byte("Flashed all the things"))
+	flasherOut.Write([]byte(fmt.Sprintln()))
+	return err //should be nil
```

Comment:
It's best to return `nil` here, take a read at this: https://golang.org/doc/faq#nil_error